srcStatic/ConsoleModuleLoader.cpp

```diff
@@ -173,3 +180,12 @@ void ConsoleModuleLoader::RunModule()
 		}
 	}
 }
+
+std::string& ConsoleModuleLoader::ModuleDirectory()
+{
+	// Function level statics are initialized on first use
+	// They are not subject to unsequenced initialization order of globals
+	// There is no order dependency on when this variable may be used
+	static std::string moduleDirectory;
```

Comment:
Is there a way to mark this as const, so the moduleDirectory cannot be changed after initialization?